dan_layer/core/src/types.rs

```diff
@@ -26,6 +26,8 @@ use tari_crypto::{
     tari_utilities::{ByteArray, ByteArrayError},
 };
 
+pub type Hash = [u8; 32];
```

Comment:
Not used?

Reply:
Don't think it's used, will remove in next PR (adding to client and some tests) if this is merged tomorrow

Reply:
Removed in this PR since there was a conflict